src/lib/viewers/doc/_docBase.scss

```diff
@@ -354,3 +336,23 @@ $thumbnail-sidebar-width: 226px;
 .bp-content.bp-is-fullscreen .bp-toggle-thumbnails-icon {
     display: none;
 }
+
+.bp-content {
+    &.bp-annotations-discoverable {
```

Comment:
Can these two line be combined into one line just like the style above? `
`.bp-content.bp-annotations-discoverable`

Reply:
So if we were to go with Conrad's approach below, I don't think I can combine them into one line.

This is what I'm thinking:

```
.bp-content {
    &.bp-annotations-discoverable {
        .textLayer {
            pointer-events: none;

            span {
                pointer-events: auto;
            }
        }

        &.bp-annotations-create--region {
            .textLayer {
                span {
                    pointer-events: none;
                }
            }
        }
    }

    // Hide the toggle thumbnails button when in fullscreen
    &.bp-is-fullscreen .bp-toggle-thumbnails-icon {
        display: none;
    }
}
```

Reply:
Yeah, this looks good to me.